index.js

```diff
@@ -89,4 +97,25 @@ module.exports = function (processors, options) {
   }
 
   return stream
+})
+
+
+function withConfigLoader(cb) {
+  return function (plugins, options) {
+    if (typeof plugins === 'undefined') {
+      return cb(postcssLoadConfig)
```

Comment:
@w0rm 👍 on exposing the vinyl file and the `cb()` approach never thought about it in that way :). Ok, I made a [gist](https://gist.github.com/michael-ciniawsky/df6f6ee1ed76c348c1849178107f08a9) tracking `postcss-load-config` integration status.


> another thing that I found in the webpack loader, is a way to specify the config location, maybe something like this would be helpful

👍 Yep that's definitely helpful, I'm working on refactoring the webpack implementation a bit already

```js
// Current
{
  loader: 'postcss-loader'
  options: { config: 'alternative/path/to/postcss.config.js' }
}
// Future
{
  loader: 'postcss-loader'
  options: { config: { path: 'alternative/path/to/postcss.config.js'  ctx: {} } }
  // options: { config: 'alternative/path/to/postcss.config.js'  ctx: {} }
}
```